exchange/bidder_test.go

```diff
@@ -141,48 +141,6 @@ func TestSingleBidder(t *testing.T) {
 	}
 }
 
-func TestRequestBidRemovesSensitiveHeaders(t *testing.T) {
```

Comment:
I recommend to keep this end to end test.

Reply:
Oops yeah that was my intention. I had temporarily removed this test and was trying to add something similar that verified the shared memory error was resolved by making multiple bid requests but that seems to be more effort than it's worth so I'll just add this back in.

Reply:
Sounds good. I wouldn't bother to try to reproduce the shared memory conflict either. It's just too hard to simulate reliably.